stubs/Pillow/PIL/ImageDraw.pyi

```diff
@@ -1,21 +1,25 @@
 from collections.abc import Container
-from typing import Any, Tuple, Union, overload
+from typing import Any, Sequence, Tuple, Union, overload
 from typing_extensions import Literal
 
 from .Image import Image
 from .ImageFont import _Font
 
 _Ink = Union[str, Tuple[int, int, int]]
```

Comment:
I think this can also be an int: https://github.com/python-pillow/Pillow/blob/master/src/_imaging.c#L506
```suggestion
_Ink = str | int | Tuple[int, int, int]
```

Reply:
Yeah, I didn't touch any of the `_Ink` stuff that was originally added in these stubs. I'll look closer.

IIRC this isn't using the pipe syntax due to mypy bugs, though.

Reply:
Oh I think you're right, we can't use `|` in aliases yet.